deepmd/dpmodel/descriptor/dpa3.py

```
@@ -0,0 +1,647 @@
+        check_version_compatibility(version, 1, 1)
+        data.pop("@class")
+        data.pop("type")
+        repflow_variable = data.pop("repflow_variable").copy()
+        type_embedding = data.pop("type_embedding")
+        data["repflow"] = RepFlowArgs(**data.pop("repflow_args"))
+        obj = cls(**data)
+        obj.type_embedding = TypeEmbedNet.deserialize(type_embedding)
+
+        # deserialize repflow
+        statistic_repflows = repflow_variable.pop("@variables")
+        env_mat = repflow_variable.pop("env_mat")
```

Ruff (0.8.2)

Local variable `env_mat` is assigned to but never used

Remove assignment to unused variable `env_mat`

(F841)